docs/r2v/data_processing_quick_start.md

```diff
@@ -141,6 +163,78 @@ objects in the cloud become more apparent:
 Once the values have been determined, place them in the 
 `config/sensor_offset.json` file for use in the pipeline.
```

Comment:
Same comment - make it clear the file is already there now

Reply:
Fixed.